diff --git a/node_modules/fonteditor-core/lib/ttf/table/cmap/parse.js b/node_modules/fonteditor-core/lib/ttf/table/cmap/parse.js
index f31f366..f8f5858 100644
--- a/node_modules/fonteditor-core/lib/ttf/table/cmap/parse.js
+++ b/node_modules/fonteditor-core/lib/ttf/table/cmap/parse.js
@@ -182,10 +182,12 @@ function readSubTable(reader, ttf, subTable, cmapOffset) {
           var numVarSelectorRecords = reader.readUint32();
           var _groups = [];
 
+          var offset = reader.offset;
           for (var _i2 = 0; _i2 < numVarSelectorRecords; _i2++) {
-            var varSelector = reader.readUint24();
-            var defaultUVSOffset = reader.readUint32();
-            var nonDefaultUVSOffset = reader.readUint32();
+            var varSelector = reader.readUint24(offset);
+            var defaultUVSOffset = reader.readUint32(offset + 3);
+            var nonDefaultUVSOffset = reader.readUint32(offset + 7);
+            offset += 11;
 
             if (defaultUVSOffset) {
               var numUnicodeValueRanges = reader.readUint32(startOffset + defaultUVSOffset);
