diff --git a/node_modules/vue-slider-component/lib/typings/index.d.ts b/node_modules/vue-slider-component/lib/typings/index.d.ts
index 016e707..89f3c8b 100644
--- a/node_modules/vue-slider-component/lib/typings/index.d.ts
+++ b/node_modules/vue-slider-component/lib/typings/index.d.ts
@@ -1,6 +1,6 @@
-/// <reference path="./global.d.ts" />
+import { DefineComponent } from "vue"
 
-import VueSlider from '../vue-slider.vue'
+declare const VueSlider: DefineComponent
 
 export interface Styles {
   [key: string]: any
