diff --git a/node_modules/vuex-persist/dist/cjs/index.js b/node_modules/vuex-persist/dist/cjs/index.js
index acd6333..9f7769d 100644
--- a/node_modules/vuex-persist/dist/cjs/index.js
+++ b/node_modules/vuex-persist/dist/cjs/index.js
@@ -150,7 +150,7 @@ class VuexPersistence {
         this.RESTORE_MUTATION = function RESTORE_MUTATION(state, savedState) {
             const mergedState = merge(state, savedState || {}, this.mergeOption);
             for (const propertyName of Object.keys(mergedState)) {
-                this._vm.$set(state, propertyName, mergedState[propertyName]);
+                state[propertyName] = mergedState[propertyName];
             }
         };
         this.asyncStorage = options.asyncStorage || false;
diff --git a/node_modules/vuex-persist/dist/esm/index.js b/node_modules/vuex-persist/dist/esm/index.js
index fca7ef5..2e4197b 100644
--- a/node_modules/vuex-persist/dist/esm/index.js
+++ b/node_modules/vuex-persist/dist/esm/index.js
@@ -144,7 +144,7 @@ class VuexPersistence {
         this.RESTORE_MUTATION = function RESTORE_MUTATION(state, savedState) {
             const mergedState = merge(state, savedState || {}, this.mergeOption);
             for (const propertyName of Object.keys(mergedState)) {
-                this._vm.$set(state, propertyName, mergedState[propertyName]);
+                state[propertyName] = mergedState[propertyName];
             }
         };
         this.asyncStorage = options.asyncStorage || false;
